content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -247,6 +251,26 @@ linux/amd64, go1.10.3, 2e322f6
 172.17.0.18:41675 - [07/Sep/2018:15:29:11 +0000] 59925 "A IN kubernetes.default.svc.cluster.local. udp 54 false 512" NOERROR qr,aa,rd,ra 106 0.000066649s
 ```
 
+### Are you in the right namespace for the service?
+
+DNS queries for services change based on namespace of the pod. DNS 
+queries that don't specify a namespace are limited to the pod's 
+namespace. 
+
+What is the namespace of the pod? What is the namespace of the 
+service being targeted by the DNS query?
+
+If the namespaces differ, the DNS query must include the namespace
+of the service.
```

Comment:
Does this content/discussion belong in the task page?
Repeated from the concept page:
```
DNS queries for services change based on namespace of the pod. DNS 
queries that don't specify a namespace are limited to the pod's 
namespace. 
```

Reply:
I've edited this down, and reduced the duplication.